examples/sidecar/deployment.yaml

```diff
@@ -0,0 +1,40 @@
+# Inspired from https://kubernetes.io/docs/tasks/access-application-cluster/communicate-containers-same-pod-shared-volume/
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: two-containers
+  namespace: kube-public
+  annotations:
+    janitor/ttl: 1m
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: two-containers
+  template:
+    metadata:
+      annotations:
+        janitor/ttl: 1m
+      labels:
+        app: two-containers
+    spec:
+      serviceAccount: kube-janitor
+      containers:
+        - name: nginx-controller
+          image: nginx
+        - name: kube-janitor-sidecar
+          image: hjacobs/kube-janitor:0.3
+          args:
+            - --interval=60
+            - --include-resources=pods,deployments,replicasets
```

Comment:
we can restrict it to `deployments` in this case (the deployment will clean up the replicaset and pods)

Reply:
when I changed it to `--include-resources=deployments`, it only seems like the deployment was deleted, and not the replicaset nor the pod